aspnetcore/blazor/javascript-interoperability/import-export-interop.md

```diff
@@ -121,6 +131,13 @@ The module name in the `[JSImport]` attribute and the call to load the module in
     "Contoso.InteropServices.JavaScript.UserMessages.CallJavaScript1")]
 ```
 
+Prefix a global function name with [`globalThis`](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/globalThis) and only pass the function name to the `[JSImport]` attribute. In the following example, [`console.log`](https://developer.mozilla.org/docs/Web/API/console/log) is prefixed with `globalThis`. The imported function is called by the C# `Log` method, which accepts a C# string message (`message`) and marshalls the C# string to a JS [`String`](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/String) for `console.log`:
```

Comment:
Or "Functions accessible on global namespace can be also imported by using `globalThis` prefix in the function name" if we want to avoid "you"

Reply:
I thought that's what I said here, including the use of the "only pass the function name" to cover the "without providing module name" aspect.

Can you provide a `suggestion` for the revision? I'm not sure how best to state it.

Reply:
Yes on avoiding "you" if possible. We usually avoid 2nd person.

Reply:
I just don't like "and only pass the function name" the negation/exclusion here. Better to say what to do rather than what not to do.  Also to set context bit more starting the sentence with imperative doesn't give me why.

Reply:
Ok ... I'll change that.

Just as an FYI outside of improving the context in this case, you'll see imperative tone used regularly to avoid "you do this/that" constructions and state verbs. *It's quite popular!* 😄 Personally, I'm less concerned about 2nd person used sparingly. I'm more concerned about the proliferation of state verbs all over the place. I don't naturally write that way and kind'a cringe when I see a lot of them. It was my upbringing no doubt ... *Mrs. Calvary, my 7th grade English teacher, is probably rolling over in her grave ⚰️ right now on this subject!* 🤣